bundles/org.openhab.binding.tesla/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -283,6 +283,11 @@
 		<label>Front Trunk</label>
 		<description>Indicates if the front trunk is opened, or open the front trunk when ON is received</description>
 	</channel-type>
+	<channel-type id="forcerefresh" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Force refresh</label>
+		<description>Forces the vehicle data to be refreshed, even when the car is not being polled to allow it to go to sleep.</description>
```

Comment:
I think that this sentence might be a little bit confusing, could you try to refine it,

Reply:
Which part do you think is confusing?